frontend/src/lib/components/ActivityLog/activityLogLogic.test.tsx

```diff
@@ -21,6 +21,7 @@ import { Provider } from 'kea'
 import React from 'react'
 import { personActivityDescriber } from 'scenes/persons/activityDescriptions'
 import { MOCK_TEAM_ID } from 'lib/api.mock'
+import { insightActivityDescriber } from 'scenes/saved-insights/activityDescriptions'
 
 const keaRender = (children: React.ReactFragment): RenderResult => render(<Provider>{children}</Provider>)
```

Comment:
This is out of scope, but `keaRender` is no longer necessary. We don't need the `<Provider>` tag starting with Kea 3.0, so if you feel like it, feel free to just use `render` instead in the code below...